tic-tac-toe/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import './Card.css'
+
+interface Props {
+  value: string
+  idx: number
+  onClick: (i: number) => void
+}
+
+const Card: React.FC<Props> = ({ value, idx, onClick }) => {
+  return (
+    <div role="button" aria-hidden className="card" onClick={() => onClick(idx)}>
```

Comment:
Explain this

Reply:
Should have used button elements. Fixing this.